### DISCUSSION TITLE
PGNet model downloaded does not perform as good as reported.

### DISCUSSION THREAD

**Discussion by lidan1** [PaddlePaddle/PaddleOCR#13516] [Q&A]
#### 问题描述 / Problem Description
I tested the PGNet downloaded from the link provided in the doc on TotalText test set, the numbers I got was lower than reported in the table.
https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_en/algorithm_e2e_pgnet_en.md#test-environment-nvidia-tesla-v100-sxm2-16gb (url)

#### 运行环境 / Runtime Environment
- OS: Linux
- Paddle: 2.4.0-rc0
- PaddleOCR: 2.4

#### 复现代码 / Reproduction Code
`python3 tools/eval.py -c configs/e2e/e2e_r50_vd_pg.yml  -o Global.checkpoints="{path/to/weights}/best_accuracy"
#### 附件 / Appendix
![Screenshot 2024-07-05 at 15 53 04](https://github.com/PaddlePaddle/PaddleOCR/assets/26124569/720eb24b-4b4e-4b28-b99c-6a4d74a13493)

**Comment by wxsIcey** [comment]
hello，我使用相同数据集对PGNet也进行了评估，发现和你结果一致，请问这个问题得到解决了吗？
